dotnet/src/SemanticKernel/Orchestration/DefaultSKContext.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.Orchestration;
+
+/// <summary>
+/// Semantic Kernel context.
+/// </summary>
+[DebuggerDisplay("{DebuggerDisplay,nq}")]
+internal sealed class DefaultSKContext : SKContext
+{
+    private CultureInfo _culture;
+    private bool _errorOccurred = false;
+    private Exception? _lastException;
+    private string _lastErrorDescription = string.Empty;
+    private IReadOnlyCollection<ModelResult> _modelResults;
+    private ContextVariables _variables;
+    private IReadOnlySkillCollection _skills;
+    private ILogger _logger;
+
+    internal static SKContext FromException(Exception exception)
+    {
+        DefaultSKContext context = new();
+        context.Fail(exception.Message, exception);
+        return context;
+    }
+
+    public override string Result => this.Variables.ToString();
+
+    public override bool ErrorOccurred => this._lastException != null;
```

Comment:
Is this property even necessary if all it's doing is seeing whether LastException is null?

Reply:
No, this was just intended to keep the current SDK consistent until we overhaul the exceptions. Then it should all go away.